src/IoTHub.Portal.Shared/Models/v1.0/UserModel.cs

```diff
@@ -0,0 +1,14 @@
+// Copyright (c) CGI France. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace IoTHub.Portal.Shared.Models.v10
+{
+
+    public class UserModel
+    {
+        public string Id { get; set; }
+        public string Email { get; set; }
+        public string Name { get; set; }
+        public string Forename { get; set; }
```

Comment:
Please use these properties in the model.
Think also to adapt the entity and table....

```suggestion
        public string Id { get; set; }
        public string Email { get; set; }
        public string Name { get; set; }
        public string GivenName { get; set; }
        public string FamilyName{ get; set; }
```

Reply:
I will do it in the next migration

Reply:
I made theses changes